### DISCUSSION TITLE
Dropdownlist in Datagrid

### DISCUSSION THREAD

**Discussion by HillMountain** [Megabit/Blazorise#3406] [Q&A]
Hi, 
can anyone point me to a working example with a dynamic loaded dropdownlist (from database) to use in a datagrid?

Thanks!

**Comment by David-Moreira** [comment]
The code for loading data from an external source can be based off this:

https://blazorise.com/docs/extensions/dropdownlist

You then just have to use the DataGrid display/edit templates to inbue the DropdownList in the DataGrid.

- If you need it to be contextual, then you can bind to the `SelectedRowChanged` and load the values accordingly.
- Or wrap `DropdownList` into your own component with `Parameters` that correspond to the filter values to load data based of each row, (The column templates provide you with the Row `TItem` for you to use).

There's alot of ways to go about this. But it depends on how you need to implement it. Let us know if this moves you in the right direction.

**Reply by HillMountain** [reply]
Thanks for quick reply!
I'm new to this type of development, do you have a link to some kind of example? I can get the dropdownlist to work separate, just not in the datagrid.

Thanks!
